lib/markright/utils.ex

```diff
@@ -77,7 +77,13 @@ defmodule Markright.Utils do
   ##############################################################################
 
   @spec to_parser_module(atom(), list()) :: atom()
-  def to_parser_module(atom, opts \\ []),
+  def to_parser_module(atom, opts \\ [])
+
+  def to_parser_module(_atom, [parser: mod] = opts) do
+    if Code.ensure_loaded?(mod), do: mod, else: opts[:fallback]
+  end
+
+  def to_parser_module(atom, opts),
     do: to_module(Markright.Parsers, atom, opts)
```

Comment:
Let’s make it possible to pass more options via `opts`.  Keywords are lists of tuples under the hood and it won’t match if `opts = [parser: :foo, fallback: :baz]`.
```suggestion
  def to_parser_module(atom, opts \\ []) do
    mod = Keyword.get(opts, :parser)
    mod =
      if mod && Code.ensure_loaded?(mod),
        do: mod,
        else: opts[:fallback]

      mod || to_module(Markright.Parsers, atom, opts)
  end
```